packages/algob/src/types.ts

```diff
@@ -463,8 +463,8 @@ export interface SSCStateSchema {
 export interface SSCOptionalFlags {
   appArgs?: Uint8Array[]
   accounts?: string[]
-  foreignApps?: string
-  foreignAssets?: string
+  foreignApps?: number[]
+  foreignAssets?: number[]
```

Comment:
i don't think `foreignAssets` type is `number[]`. 
i think it's a array of object with fields.(https://developer.algorand.org/docs/reference/teal/opcodes/#asset_params_get)

Reply:
As per sdk it's an array of 'indexes'. 
![image](https://user-images.githubusercontent.com/33264364/101929072-63608780-3bfc-11eb-9d44-6da44003b61a.png)

Reply:
ok, Transaction reference says `address`(https://developer.algorand.org/docs/reference/transactions/#asset-transfer-transaction). but this seems right.